### PR TITLE
Trip overview

### LINKED ISSUE #10: Dashboard : Overview of Trips
UI for loading all of the user's trips. 
image

### PR DESCRIPTION
**Pull request by aamare1001**
Adds in the Trip Cards for the dashboard
Trips Dashboard mockup

Currently is static, does not add multiple per trips for user(just the UI Component)

### CODE REVIEW

File: app/app.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ import { GoogleSignin } from "@react-native-google-signin/google-signin"
 import { WEB_CLIENT_ID, TWITTER_APP_ID, TWITTER_KEY, TWITTER_SECRET } from "@env"
 import { Settings } from "react-native-fbsdk-next"
```
Comment (Victoria-Williamson) [inline app/app.tsx:27]: Remove unused imports

File: app/components/TripCard.tsx [hunk 2]
```diff
@@ -1,10 +1,87 @@
-import { View, Text } from "react-native"
+import { View, ImageBackground, ViewStyle, StyleProp } from "react-native"
 import React from "react"
+import { Card } from "./Card"
+import { Icon } from "./Icon"
+import { colors, typography } from "../theme"
+import Divider from "./Divider"
+import { Text } from "./Text"
```
Comment (Victoria-Williamson) [inline app/components/TripCard.tsx:4]: I beleive you can also run ``yarn lint`` to check these ^^. But I can make a pre-commit hook so it shows up there

File: app/components/TripCard.tsx [hunk 3]
```diff
@@ -1,10 +1,87 @@
...
+              size="sm"
+              style={{
+                textAlign: "right",
+              }}
+            />
+          ) : (
+            <Text text={`${getTime(trip.duration.start)}`} preset="subheading" size="sm" />
+          )}
+
+          {/* <Icon icon="personSmall" style={{left:300}}/> */}
```
Comment (Victoria-Williamson) [inline app/components/TripCard.tsx]: What was this for? Was this an old version of the avatars?

File: app/screens/TripHome.tsx [hunk 4]
```diff
@@ -1,26 +1,99 @@
...
 import { useAuth } from "../models/hooks"
+import TripCard from "../components/TripCard"
 
 type TripHomeProps = AppStackScreenProps<"TripHome">
 export const TripHome: FC<TripHomeProps> = observer(function TripHomeScreen() {
   const { doLogout } = useAuth()
+  const background = {
+    back: require("../../assets/images/gradientBg.png"),
+  }
+  const exTrips = [
```
Comment (Victoria-Williamson) [inline app/screens/TripHome.tsx]: I would rename these trips, and when u display the trips below, do trips.map() and set up a function to separate current and upcoming trips so when we connect this to the data response from backend ( which will come in chronological order ) we just need to set this variable up to store the fetch response.

File: types/auth.ts [hunk 5]
```diff
@@ -7,3 +7,15 @@ export interface User {
   medicalInfo: Array<string>
   allergies: Array<string>
 }
+
+export type Trip = {
```
Comment (Victoria-Williamson) [inline types/auth.ts]: Sorry I forgot to mention I think. Move this to another file called ``trip.ts``

**Review by Victoria-Williamson — CHANGES_REQUESTED** [review]

File: app/components/TripCard.tsx (hunk 3 above)
Comment (aamare1001) [reply app/components/TripCard.tsx]: Yes, it's scrap thank u!

**Review by Victoria-Williamson — APPROVED** [review]

**Pull request merged**